docs/end-user-guide/using-jira-commands.md

```diff
@@ -14,17 +14,17 @@ The available commands are listed below.
 
 **Note:** [setting] can be `notifications` and [value] can be `on` or `off`
 
-* `/jira help` - Launch the Jira plugin command line help syntax
-* `/jira info` - Display information about the current user and the Jira plugin
-* `/jira instance list` - List installed Jira instances
-* `/jira connect [jiraURL]` - Connect your Mattermost account to your Jira account
-* `/jira disconnect [jiraURL]` - Disconnect your Mattermost account from your Jira account
-* `/jira [issue] assign [issue-key] [assignee]` - Change the assignee of a Jira issue
-* `/jira [issue] create [text]` - Create a new Issue with 'text' inserted into the description field
-* `/jira [issue] transition [issue-key] [state]` - Change the state of a Jira issue
-* `/jira [issue] unassign [issue-key]` - Unassign the Jira issue
-* `/jira [issue] view [issue-key]` - View the details of a specific Jira issue
-* `/jira instance settings [setting] [value]` - Update your user settings
+- `/jira help` - Launch the Jira plugin command line help syntax
+- `/jira info` - Display information about the current user and the Jira plugin
+- `/jira instance list` - List installed Jira instances
+- `/jira connect [jiraURL]` - Connect your Mattermost account to your Jira account
+- `/jira disconnect [jiraURL]` - Disconnect your Mattermost account from your Jira account
+- `/jira [issue] assign [issue-key] [assignee]` - Change the assignee of a Jira issue
+- `/jira [issue] create [text]` - Create a new Issue with 'text' inserted into the description field
+- `/jira [issue] transition [issue-key] [state]` - Change the state of a Jira issue
+- `/jira [issue] unassign [issue-key]` - Unassign the Jira issue
+- `/jira [issue] view [issue-key]` - View the details of a specific Jira issue
+- `/jira instance settings [setting] [value]` - Update your user settings
```

Comment:
Sure. That makes sense to me. I can revert these changes.
Apologies since my VS Code editor somehow "edited" these files automatically. (probably through one of the extensions).

Reply:
For sure let's revert the changes

Reply:
Apologies for delay. Let me revert the unnecessary changes and get back.

Reply:
2/5 I don't mind to have the asterisks to dashes migration in this PR. It's a "docs" PR and hence can contains any docs updates.